examples/component-composition/app/components/list-view.js

```diff
@@ -1,23 +1,26 @@
-import Component from '@ember/component';
```

Comment:
Let's default to having components in their own directories (`app/components/list-view/component.js`) – that's what I tell people to do in the slides as well now.

Reply:
Moved components into their own directories